clients/drcachesim/tests/signal_invariants.c

```diff
@@ -194,7 +194,7 @@ GLOBAL_LABEL(FUNCNAME:)
         prefetcht2 [3]
         nop
         nop
-        ud2a
+        ud2
```

Comment:
(There was some general community rename at some point as DR got its names ud2a + ud2b from somewhere, and even today assemblers are happy with `ud2a`: not sure assemblers take `ud2b`.)